test/app/routes/tracks_test.rb

```diff
@@ -16,6 +16,14 @@ def test_all_the_tracks
     Approvals.verify(tracks, name: 'get_tracks')
   end
 
+  def test_all_the_todos
+    get '/tracks'
+    tracks = JSON.parse(last_response.body)['tracks'].find { |track|
+      track['slug'] == 'clojure'
+    }['todo']
```

Comment:
Try sticking `.sort` on the end of this line, then commit and push to the lax remote to update the branch so that the build can try again.